componets/modal/modal.html

```diff
@@ -0,0 +1,228 @@
+						<li><a href="../buttons/buttons.html">Buttons</a></li>
+						<li><a href="../cards/cards.html">Cards</a></li>
+						<li><a href="../Images/images.html">Images</a></li>
+						<li><a href="../input/input.html">Input</a></li>
+						<li>
+							<a href="../text_Utilities/textUtilities.html">Text Utilities</a>
+						</li>
+						<li><a href="../lists/lists.html">Lists</a></li>
+						<li><a href="../ratings/ratings.html">Ratings</a></li>
+						<li><a href="./modal.html">Modal</a></li>
+						<li><a href="../snackBar/snackBar.html">Snack Bar</a></li>
+						<li><a href="../grid/grid.html">Grid</a></li>
+						<li><a href="../navigation/navigation.html">Navigation</a></li>
+						<li><a href="../slider/slider">Slider</a></li>
+					</ul>
+				</div>
+			</aside>
+			<main>
+				<h1 class="component__heading" id="modal">Modal</h1>
+				<p class="component__description">
+					Modal is mainly used to show some notification to user or take some
+					input.
+				</p>
+				<div class="component_types" id="simpleModal">
+					<h2 class="component__name">Simple Modal</h2>
+					<p class="component__description">
+						use class <code>modal</code> to form a simple notification modal to
+						confirm something from user.
+					</p>
+					<div class="components__example">
+						<div class="modal">
+							<div class="modal__dismiss">
+								<h2 class="modal__title">Delete item</h2>
+								<i class="far fa-times-circle"></i>
+							</div>
+							<p class="modal__text">Are you sure you want to delete</p>
+							<div class="modal__buttons">
+								<button class="btn">Cancel</button>
+								<button class="btn btn-primary">Delete</button>
+							</div>
+						</div>
+					</div>
+					<div class="component_code">
+						<iframe
+							src="https://carbon.now.sh/embed?bg=rgba%280%2C0%2C0%2C1%29&t=material&wt=none&l=javascript&width=680&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=0px&ph=0px&ln=true&fl=1&fm=Hack&fs=14px&lh=144%25&si=false&es=2x&wm=false&code=%2509%2509%2509%253Cdiv%2520class%253D%2522modal%2522%253E%250A%2509%2509%2509%2509%253Cdiv%2520class%253D%2522modal__dismiss%2522%253E%250A%2509%2509%2509%2509%2509%253Ch2%2520class%253D%2522modal__title%2522%253EDelete%2520item%253C%252Fh2%253E%250A%2509%2509%2509%2509%2509%253Ci%2520class%253D%2522far%2520fa-times-circle%2522%253E%253C%252Fi%253E%250A%2509%2509%2509%2509%253C%252Fdiv%253E%250A%2509%2509%2509%2509%2520%2520%2520%2520%2520%253Cp%2520class%253D%2522modal__text%2522%253EAre%2520you%2520sure%2520you%2520want%2520to%2520delete%253C%252Fp%253E%250A%2509%2509%2509%2509%253Cdiv%2520class%253D%2522modal__buttons%2522%253E%250A%2509%2509%2509%2509%2509%2520%253Cbutton%2520class%253D%2522btn%2522%253ECancel%253C%252Fbutton%253E%250A%2509%2509%2509%2509%2509%253Cbutton%2520class%253D%2522btn%2520btn-primary%2522%253EDelete%253C%252Fbutton%253E%250A%2509%2509%2509%2509%253C%252Fdiv%253E%250A%2509%2509%2509%253C%252Fdiv%253E"
+							style="min-height: 280px"
```

Comment:
Suggestion: avoid using inline styles, you can write this style in css file for all iframes

Reply:
yes i tried that but different iframes need different height so I am using inline styles only for  iframes